src/runners/express.js

```diff
@@ -17,7 +18,7 @@ const {
  */
 function createRunner(req, startTime) {
     const expressEvent = new event.Event([
-        `express-${uuid4()}`,
+        extractEpsagonHeader(req.headers) || `express-${uuid4()}`,
```

Comment:
maybe use it as another field in the metadata? will be more explicit and easier to port later

Reply:
(everywhere)

Reply:
👍